tox.ini

```diff
@@ -18,6 +18,7 @@ commands =
 sitepackages = False
 commands =
   flake8 {posargs}
+  black --check {posargs} .
 
```

Comment:
Can you add a `black` job here too so people can just run `tox -eblack` and get it to take care of auto formatting. Something like:
```ini
[testenv:black]
envdir = .tox/pep8
commands = black {posargs} stestr tools docs setup.py
```

Reply:
done